### ISSUE TITLE
Would you please share the schedule of filter functions

### ISSUE THREAD

**Issue by The-Mountain** [schibsted/jslt#141]
I see the filter function is in your ideas list, would you please share the detail schedule when will fininsh filter macro implementation, it's really a very useful function to get a specific value in an arrays, and it's an very familiar use case for common businesses, and it's very easy to do this in other language like JSONPath

**Comment by The-Mountain** [comment]
I mean the function to get a specific value with condition in an array, similar with below functions in JsonPath
$.store.book[?(@.category == 'reference')].author
thanks!

**Comment by larsga** [comment]
Yes, I've been thinking that this is a feature the language needs. It's related to being able to get key `x` from all objects in an array, without having to write it like:

`[for (.path.to.array) .x]`

which is clumsy to combine with further navigation.

For transformation this is not a very common use case, but for querying it is.

I'm hesitant to give you a schedule, but I'd like to start working on this. The first step would be to put together a proper proposal.
